src/components/ContinuationIn/SummaryBusinessPreviousJurisdiction.vue

```diff
@@ -286,20 +295,41 @@ article:not(:last-child) {
   padding-bottom: 0;
 }
 
-// vertically align file buttons with label
-.file-buttons {
-  margin-top: -6px !important;
+.download-authorization-btn,
+.download-affidavit-btn {
+  // set button height for multi-line filenames
+  height: auto !important;
+
+  .v-icon {
+    // minimum width of icons (to align them)
+    min-width: 24px;
+  }
+  span {
+    // make button text larger than default
+    font-size: $px-16;
+    // wrap long filenames
+    text-wrap: wrap;
+  }
 }
 
-.download-affidavit-btn,
-.download-authorization-btn {
-  // nudge icon down a bit to line up with text
+.approved-checkmark {
   .v-icon {
-    margin-top: 2px;
+    // minimum width of icons (to align them)
+    min-width: 24px;
+    // nudge icon up a bit to line up with text
+    margin-top: -1px;
   }
-  // make button text larger than default
   span {
+    // make button text larger than default
     font-size: $px-16;
   }
 }
+
+// "md" breakpoint
+@media (min-width: 960px) {
+  .approved-checkmark .v-icon {
+    // nudge icon down a bit to line up with text
+    margin-top: 1px;
+  }
+}
```

Comment:
I got fancy here so that the checkmark vertically aligns with the text while narrow and full-width.

![image](https://github.com/user-attachments/assets/b84bba48-4185-44b0-986a-04265d8c5cd7)

![image](https://github.com/user-attachments/assets/4ebcaabd-f2f2-4b58-bc0d-666cf9f73047)